src/layman/layer/prime_db_schema/tasks.py

```diff
@@ -0,0 +1,34 @@
+from celery.utils.log import get_task_logger
+
+from layman.celery import AbortedException
+from layman import celery_app
+from ..db import table as db_table
+from .bbox import set_bbox, delete_layer
+
+
+logger = get_task_logger(__name__)
+
+
+def refresh_bbox_needed(username, layername, task_options):
+    return True
+
+
+@celery_app.task(
+    name='layman.layer.prime_db_schema.bbox.refresh',
+    bind=True,
+    base=celery_app.AbortableTask
+)
+def refresh_bbox(
+        self,
+        username,
+        layername,
+):
+    if self.is_aborted():
+        raise AbortedException
+
+    bbox = db_table.get_layer_bbox(username, layername)
+    set_bbox(username, layername, bbox, )
+
+    if self.is_aborted():
+        delete_layer(username, layername)
```

Comment:
Layer should not be deleted from prime db schema at this moment. Either do nothing, or set bbox column to empty bbox.